2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_setup.py:_flush():76] Configure stats pid to 14483
2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_b', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'sweep_id': '9lkzkcbg', 'run_id': 'ihgzkx10', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-9lkzkcbg/config-ihgzkx10.yaml'}
2022-08-01 18:08:03,352 WARNING Thread-3  :14483 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_180803-ihgzkx10/logs/debug.log
2022-08-01 18:08:03,352 INFO    Thread-3  :14483 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_180803-ihgzkx10/logs/debug-internal.log
2022-08-01 18:08:03,356 INFO    Thread-3  :14483 [wandb_init.py:init():495] calling init triggers
2022-08-01 18:08:03,356 INFO    Thread-3  :14483 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 128, 'IMS_PER_BATCH': 8, 'iteration': 300, 'learning_rate': 0.0003363527377317388, 'model': 'COCO-Detection/faster_rcnn_R_101_DC5_3x.yaml'}
config: {}
2022-08-01 18:08:03,359 INFO    Thread-3  :14483 [wandb_init.py:init():548] starting backend
2022-08-01 18:08:03,359 INFO    Thread-3  :14483 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 18:08:03,378 INFO    Thread-3  :14483 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 18:08:03,383 INFO    Thread-3  :14483 [backend.py:ensure_launched():222] started backend process with pid: 14529
2022-08-01 18:08:03,385 INFO    Thread-3  :14483 [wandb_init.py:init():558] backend started and connected
2022-08-01 18:08:03,398 INFO    Thread-3  :14483 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 128, 'IMS_PER_BATCH': 8, 'iteration': 300, 'learning_rate': 0.0003363527377317388, 'model': 'COCO-Detection/faster_rcnn_R_101_DC5_3x.yaml'}
2022-08-01 18:08:03,401 INFO    Thread-3  :14483 [wandb_init.py:init():634] updated telemetry
2022-08-01 18:08:03,403 INFO    Thread-3  :14483 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 18:08:04,178 INFO    Thread-3  :14483 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 18:08:04,212 INFO    Thread-3  :14483 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 18:08:04,212 INFO    Thread-3  :14483 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 18:08:05,323 INFO    Thread-3  :14483 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 18:08:05,328 INFO    Thread-3  :14483 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 18:08:05,329 INFO    Thread-3  :14483 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 18:08:05,329 ERROR   Thread-3  :14483 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 18:08:05,331 INFO    Thread-3  :14483 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 18:08:15,075 INFO    Thread-3  :14483 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_b/ihgzkx10
2022-08-01 18:08:15,076 INFO    Thread-3  :14483 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-01 18:08:15,081 INFO    Thread-3  :14483 [wandb_run.py:_restore():1834] restore
2022-08-01 18:08:15,085 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 18:08:15,193 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 18:08:15,463 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 18:08:16,209 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5398
}

2022-08-01 18:08:16,311 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6039
}

2022-08-01 18:08:16,414 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6039
}

2022-08-01 18:08:16,516 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 6039
}

2022-08-01 18:08:16,618 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5924
  total_bytes: 6039
}

2022-08-01 18:08:16,721 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-01 18:08:16,823 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-01 18:08:16,925 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-01 18:08:17,027 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-01 18:08:17,129 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-01 18:08:17,662 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}

2022-08-01 18:08:18,208 INFO    Thread-3  :14483 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6039
  total_bytes: 6039
}
local_info {
}

2022-08-01 18:08:19,352 INFO    Thread-3  :14483 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-01 18:08:19,353 INFO    Thread-3  :14483 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-01 18:08:19,358 INFO    Thread-3  :14483 [wandb_run.py:_footer_sync_info():3055] logging synced files
